### PR TITLE
LRDOCS-8546 SME edits

### PR DESCRIPTION
**Pull request by lipusz**
https://issues.liferay.com/browse/LRDOCS-8546

Hey Russ,

See my changes. Don't forget to update the screenshot. :-)

Note: I've replaced the occurrences where you linked the PDF version with the Help Center page that is accessible by anyone without signing in.

I'm sending this back against your branch, not master to keep the diff clean.

Cheers,
Tibor

### CODE REVIEW

**Comment by rbohl** [comment]
Merged Locally. Thanks!

File: docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/securing-elasticsearch.md [hunk 1]
```diff
@@ -178,10 +184,10 @@ xpack.security.http.ssl.verification_mode: certificate
 
 ### Configure Liferay's Secure Connection to Elasticsearch
 
-The Elasticsearch connector bundled with Liferay 7.3 includes X-Pack Security support.
+The Elasticsearch connector bundled with Liferay 7.3 and the [Liferay Connector to Elasticsearch 7](https://web.liferay.com/marketplace/-/mp/application/170390307) available as a drop-in replacement for DXP 7.1 and 7.2 includes X-Pack Security support.
```
Comment (rbohl) [inline docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/securing-elasticsearch.md:187]: Can I just say this to simplify?

> All Liferay connectors to Elasticsearch 7 include X-Pack Security Support.

@lipusz

**Comment by lipusz** [comment]
Great idea! We don't need to capitalize "Support" I believe.

Sent from mobile
--
Product Supportability Manager & SME for Search & Security

**Comment by rbohl** [comment]
Forwarded: https://github.com/sez11a/liferay-learn/pull/264

**Pull request closed without merging**